### DISCUSSION TITLE
Plugins with combined types

### DISCUSSION THREAD

**Discussion by Gedeen** [WinMerge/winmerge#949] [Q&A] [answered]
Until version 2.16.12 a plugin was one of type Unpacker, Prediffer or editor script. In the latest version it seems that a plugin can be an Unpacker and an editor script or some other combination. How can this be achieved ?

I.e. the MakeLower plugin can be used as an Unpacker and as an editor script, but in the MergePlugins folder I can only find this in the editor addin.sct script where it says: get_PluginEvent = "EDITOR_SCRIPT". So how can this also be an Unpacker plugin ?

Is there any information about this ?

Greetings

**Comment by sdottaka** [comment, marked as answer]
When "GenerateUnpacker" is specified in the ExtendedProperties property, it will be registered as an Unpacker plugin.

https://github.com/WinMerge/winmerge/blob/master/Plugins/dlls/editor%20addin.sct#L64

**Reply by Gedeen** [reply]
Clear. Thanks.
